.github/workflows/build-ci.yml

```diff
@@ -11,6 +11,8 @@ jobs:
         name: "PHP ${{ matrix.php }} Laravel ${{ matrix.laravel }} MongoDB ${{ matrix.mongodb }} ${{ matrix.mode }}"
 
         strategy:
+            # Tests with Atlas fail randomly
```

Comment:
Consider making an open ticket with the "Build Failure" type so you don't lose track of this. Assuming it's something that can be researched further down the line.

Reply:
[PHPORM-287](https://jira.mongodb.org/browse/PHPORM-287)